### ISSUE TITLE
Default watchers must be renamed to use “default” naming

### ISSUE THREAD

**Issue by JayDi85** [magefree/mage#12345]
Where are many default watchers (added to game by default on init; commander and tiny leaders also have default watchers).

In most use cases it’s a hidden watchers (used inside ability/effect/hint) and dev don’t know is it required or not without ability code view.

It can be useful to rename all that watchers to “xxxDefaultWatcher”.
![IMG_0443](https://github.com/magefree/mage/assets/8344157/b427e919-119f-4a6a-b4ea-b6c81ad53f9a)

**Comment by xenohedron** [comment]
Good idea. If renaming, I'd also take the opportunity to put them in a separate package `default` instead of `common`.

Let's wait until the bulk of MH3 is implemented and merged though.
